tests/emodulus_calibration/test_prediction.py

```diff
@@ -0,0 +1,22 @@
+from lebedigital.calibration.posterior_predictive import perform_prediction_three_point_bending
+import pytest
+import numpy as np
+import os
+from pathlib import Path
+
+
+def test_prediction():
+    print("Prediction test started")
+
+    # check if KG is present
+    kg_path = "prediction_data/prediction_kg.owl"
```

Comment:
I'm confused here, why do you load an ontology? In addition, the unit test should test the individual function, which here would be IMO e.g. the posterior samples of the parameters, not a knowledge graph.